### PR TITLE
Missing Zephyr keys updated for guest_invitation_ui_spec.js

### PR DESCRIPTION
**Pull request by jgilliam17**
#### Summary
Updating and/or adding missing Zephyr keys 

#### Related Pull Requests

- Has server changes (please link here)
- Has mobile changes (please link here)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/integration/bot_accounts/sidebar_display_spec.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ describe('Bot accounts', () => {
         cy.apiAdminLogin();
     });
 
-    it('MM-T1836_2 Bot accounts display', () => {
+    it('MM-T1836 Bot accounts display', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/bot_accounts/sidebar_display_spec.js:69]: Nice catch! Thanks @jgilliam17!

**Review by saturninoabril — APPROVED** [review]
Great, thanks @jgilliam17!

**Review by furqanmlk — APPROVED** [review]
Thanks @jgilliam17

**Pull request merged**